src/Kros.AspNetCore/Environments/Environments.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Microsoft.AspNetCore.Hosting
+{
+    /// <summary>
+    /// Valid environments
+    /// </summary>
+    public static class Environments
+    {
+        public const string Development = "Development";
```

Comment:
I think this will cause a warning. We have rule that public members must have a comment.